### DISCUSSION TITLE
What is the equivalent to exception() from react-ga?

### DISCUSSION THREAD

**Discussion by DevEsteves** [codler/react-ga4#36] [Q&A] [answered]
In the package [react-ga](https://github.com/react-ga/react-ga), there's the method exception(), I didn't find it here. 

How can i perform something such as the following example using react-ga4?

`export const logException = (description = '', fatal = false) => {
  if (description) {
    ReactGA.exception({ description, fatal })
  }
}`

**Comment by codler** [comment, marked as answer]
https://github.com/codler/react-ga4/issues/40#issuecomment-5462111835
```js
ReactGA.gtag('event', 'exception', {
  description: error.message,
  fatal: false
});
